airflow/migrations/versions/0130_2_8_0_make_connection_login_password_text.py

```diff
@@ -0,0 +1,60 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Make connection login and password text
+
+Revision ID: 4e4a4b9a55c0
+Revises: 375a816bbbf4
+Create Date: 2023-09-14 10:04:20.416324
+
+"""
+from __future__ import annotations
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "4e4a4b9a55c0"
+down_revision = "375a816bbbf4"
+branch_labels = None
+depends_on = None
+airflow_version = "2.8.0"
+
+TABLE_NAME = "connection"
+
+
+def upgrade():
+    """Apply Make connection login and password text"""
+    with op.batch_alter_table(TABLE_NAME) as batch_op:
+        batch_op.alter_column(
+            "login", existing_type=sa.VARCHAR(length=500), type_=sa.TEXT(), existing_nullable=True
```

Comment:
These types should match the declared types in models.

Reply:
You're right. They should be matching now. Could you take another look?